src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const lessThanThreeDays = 3;
+  const moreOrEqualsThanSevenDays = 7;
```

Comment:
```suggestion
  const smallDiscountPeriod = 3;
  const bigDiscountPeriod = 7;
```
Присоединяюсь к @varseniuk , значение переменной не должно упоминаться в ее названии

Reply:
Окай, змінюєм. ;)